src/test/ui/abi/segfault-no-out-of-stack.rs

```diff
@@ -3,6 +3,7 @@
 #![allow(unused_imports)]
 // ignore-emscripten can't run commands
 // ignore-sgx no processes
+// ignore-fuchsia must translate zircon signal to SIGSEGV/SIGBUS, FIXME (#109976)
```

Comment:
This test has a handler for not(unix); we can use that one for Fuchsia without disabling the test entirely.

Reply:
A previous version changed those lines into:
`#[cfg(all(unix, not(target_os = "fuchsia")))]`
`#[cfg(any(not(unix), target_os = "fuchsia"))]`
Is this how I should approach the change?

Reply:
Yeah I think that's right but you could also leave a FIXME with a link to the same issue you're using for the other tests.

Reply:
I think that manually adding Fuchsia to the `#[cfg(not(unix))]` case may cause us to forget that this test does not actually pass for Fuchsia. We have to either support this functionality or remove Fuchsia from `#[cfg(unix)]`, upon which this test will pass with no modifications.

Reply:
To clarify, the test "passes" in the sense that it does not have a successful status `!status.success()`, but "fails" in the sense that the signal is not what is expected (SIGSEGV/SIGBUS). I agree that this comes down to whether this is expected behavior or not for Posix-lite